Gazebo.md

```diff
@@ -1,122 +1,122 @@
-| [cmake-main][cmake-main-repo] | [![Build Status][cmake-main-Linux-badge]][cmake-main-Linux] | [![Build Status][cmake-main-Windows-badge]][cmake-main-Windows] | [![Build Status][cmake-main-Homebrew-badge]][cmake-main-Homebrew] | [![Build Status][gz-cmake5-Nightlies jammy-badge]][gz-cmake5-Nightlies jammy] | [![Build Status][gz-cmake5-Nightlies noble-badge]][gz-cmake5-Nightlies noble] |
-| [tools-main][tools-main-repo] | [![Build Status][tools-main-Linux-badge]][tools-main-Linux] | [![Build Status][tools-main-Windows-badge]][tools-main-Windows] | [![Build Status][tools-main-Homebrew-badge]][tools-main-Homebrew] | [![Build Status][gz-tools3-Nightlies jammy-badge]][gz-tools3-Nightlies jammy] | [![Build Status][gz-tools3-Nightlies noble-badge]][gz-tools3-Nightlies noble] |
-| [utils-main][utils-main-repo] | [![Build Status][utils-main-Linux-badge]][utils-main-Linux] | [![Build Status][utils-main-Windows-badge]][utils-main-Windows] | [![Build Status][utils-main-Homebrew-badge]][utils-main-Homebrew] | [![Build Status][gz-utils4-Nightlies jammy-badge]][gz-utils4-Nightlies jammy] | [![Build Status][gz-utils4-Nightlies noble-badge]][gz-utils4-Nightlies noble] |
-| [sdformat-main][sdformat-main-repo] | [![Build Status][sdformat-main-Linux-badge]][sdformat-main-Linux] | [![Build Status][sdformat-main-Windows-badge]][sdformat-main-Windows] | [![Build Status][sdformat-main-Homebrew-badge]][sdformat-main-Homebrew] | [![Build Status][sdformat16-Nightlies jammy-badge]][sdformat16-Nightlies jammy] | [![Build Status][sdformat16-Nightlies noble-badge]][sdformat16-Nightlies noble] |
+| | Linux | Windows | Homebrew |
+|-|-|-|-|
+| [sim-main][sim-main-repo] | [![Build Status][sim-main-Linux-badge]][sim-main-Linux] | [![Build Status][sim-main-Windows-badge]][sim-main-Windows] | [![Build Status][sim-main-Homebrew-badge]][sim-main-Homebrew] | [![Build Status][sim-main-Nightlies noble-badge]][sim-main-Nightlies noble] |
+| [gui-main][gui-main-repo] | [![Build Status][gui-main-Linux-badge]][gui-main-Linux] | [![Build Status][gui-main-Windows-badge]][gui-main-Windows] | [![Build Status][gui-main-Homebrew-badge]][gui-main-Homebrew] | [![Build Status][gui-main-Nightlies noble-badge]][gui-main-Nightlies noble] |
+| [launch-main][launch-main-repo] | [![Build Status][launch-main-Linux-badge]][launch-main-Linux] | [![Build Status][launch-main-Windows-badge]][launch-main-Windows] | [![Build Status][launch-main-Homebrew-badge]][launch-main-Homebrew] | [![Build Status][launch-main-Nightlies noble-badge]][launch-main-Nightlies noble] |
+| [rendering-main][rendering-main-repo] | [![Build Status][rendering-main-Linux-badge]][rendering-main-Linux] | [![Build Status][rendering-main-Windows-badge]][rendering-main-Windows] | [![Build Status][rendering-main-Homebrew-badge]][rendering-main-Homebrew] | [![Build Status][rendering-main-Nightlies noble-badge]][rendering-main-Nightlies noble] |
```

Comment:
Is leaving a space in the tag name working? I would be expecting the names to be `rendering-main-nightlies-noble` instead of `rendering-main-nightlies noble`.

If it's working, I don't mind it remains that way